public/js/replay.js

```diff
@@ -151,16 +324,20 @@ function Replay (params) {
    *  Recursively plays through the rest of the operations.
    */
   this.slide = function () {
-    if (instance.current_v >= instance.operations.length) {
+    if (instance.current_v >= instance.operations.length || (instance.operations[instance.current_v].flagged && !this.at_flag)) {
+      console.log(this.flagged);
+      this.at_flag = true;
       $('#start-replay-button').toggleClass('active');
       $('#start-replay-button').find('span.glyphicon').toggleClass('glyphicon-pause').toggleClass('glyphicon-play');
+
+      this.stopped = true;
       stop = false;
```

Comment:
This is kind of confusing... we set `stop` to false, and `this.stopped` to true.  Can you add a comment explaining what is happening there?